process/block/sovereignChainBlock_test.go

```diff
@@ -883,3 +893,269 @@ func TestSovereignShardProcessor_CreateBlock(t *testing.T) {
 		require.Equal(t, expectedBusyIdleSequencePerCall, busyIdleCalled)
 	})
 }
+
+func TestSovereignShardProcessor_ProcessBlock(t *testing.T) {
+	t.Parallel()
+
+	t.Run("nil header handler should error", func(t *testing.T) {
```

Comment:
Here and everywhere else

```suggestion
	t.Run("nil header handler, should error", func(t *testing.T) {
```